Comp2024/src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,45 @@
+//
+// Intake Subystem - takes in Notes and delivers them to the Shooter
+//
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+//
+// Intake subsystem class
+//
+public class Climber extends SubsystemBase
+{
+  // Member objects
+  private final TalonFX m_climberL = new TalonFX(0);
```

Comment:
We'll need to grab the CANID constants from Zara as the next step, but this is OK to get this checked in.